### ISSUE TITLE
I just used 4times,but it says "httpx.ConnectTimeout: timed out"

### ISSUE THREAD

**Issue by zhuxiaodics6** [OwO-Network/PyDeepLX#16]
httpx.ConnectTimeout: timed out

**Comment by missuo** [comment]
```bash
pip install pydeeplx==1.0.6
```

try again.

**Comment by zhuxiaodics6** [comment]
![image](https://github.com/OwO-Network/PyDeepLX/assets/79239071/32765f9a-bc28-460b-863e-45eef5a48d4d)
it doest work

**Comment by zhuxiaodics6** [comment]
does it support to translate list string

**Comment by missuo** [comment]
> does it support to translate list string

I noticed that the error reported was a timeout. Please ensure that your local network can access Deep L

**Comment by zhuxiaodics6** [comment]
I can use the Deepl as usual,but if I use deeplx it will be wrong

**Comment by zhuxiaodics6** [comment]
![image](https://github.com/OwO-Network/PyDeepLX/assets/79239071/1a6c4b83-7a29-4a6f-8b86-13af81761882)
this is my code
imaybe, i request too fast

**Comment by zhuxiaodics6** [comment]
![image](https://github.com/OwO-Network/PyDeepLX/assets/79239071/e593ee92-b4ef-49cf-97bc-b37720ba7767)

**Comment by zhuxiaodics6** [comment]
Can I translate list of string like the official api
I want to translate a long doc
i first try to put words in one string , it says 413
then i split it into some strings and sleep 1sec in between, but it says"httpx.ConnectTimeout: _ssl.c:1074: The handshake operation timed out"or"PyDeepLX.PyDeepLX.TooManyRequestsException: PyDeepLX Error: Too many requests, your IP has been blocked by DeepL temporarily, please don't request it frequently in a short time." after I use the api for 5times
thanks for your help

**Comment by xitex** [comment]
Hi! Where can we find more valid user agents? Thanks!
